src/components/date-picker-month-header/date-picker-month-header.tsx

```diff
@@ -152,14 +151,7 @@ export class DatePickerMonthHeader {
               type="text"
               value={localizedYear}
             />
-            {suffix && (
-              <span class="suffix">
-                <span aria-hidden="true" class="suffix__invisible">
```

Comment:
Wasn't sure why we have the `span` element with `visibility: hidden`

Reply:
Looks like this span was here to hide it from screen readers. Seems like that functionality is removed here. Are we sure we can no longer hide this text from screen readers?

Reply:
The `span` element here is masking the `input` element which displays the year. If we want to hide the input element then this needs to be restored. Might be missing some context here.  @geospatialem  WDYT?

Reply:
I'm hearing some invalid entries in the demos with NVDA. Wondering if the `aria-hidden` prop is used to hide those entries? Without this change, its registering as blank in the Arabic demos.

Reply:
The comment above is when the screen reader is set to English. Once NVDA's language is set to the appropriate locale, the year number, or "year" text is read back to the screen reader. Great work, @anveshmekala! 👍🏻